code/game/objects/structures/watercloset.dm

```diff
@@ -108,11 +108,37 @@
 						GM.apply_damage(5, BRUTE, BODY_ZONE_HEAD)
 			else
 				to_chat(user, "<span class='warning'>You need a tighter grip!</span>")
+	if(istype(I, /obj/item/flamethrower))
+		var/obj/item/flamethrower/big_lighter = I
+		if(!big_lighter.lit)
+			to_chat(user, "<span_class = 'warning'>The flamethrower isn't lit!</span>")
+			return
+		big_lighter.default_ignite(loc, 0.01)
+		if(!cistern) //Just changes what message you get, since fire_act handles the open cistern too.
+			user.visible_message("<span class='warning'>[user] torches the contents of the top of the toilet with [big_lighter]!</span>", "<span class='warning'>You torch the top of the toilet with [big_lighter]! Whoops.</span>")
+			return
+
+		user.visible_message("<span class='notice'>[user] torches the contents of the cistern with [big_lighter]!</span>", "<span class='notice'>You torch the conents of the cistern with [big_lighter]!</span>")
```

Comment:
```suggestion
		user.visible_message("<span class='notice'>[user] torches the contents of the cistern with [big_lighter]!</span>", "<span class='notice'>You torch the contents of the cistern with [big_lighter]!</span>")
```

Reply:
I can't be mad at other people for fixing my silly mistakes for me, thanks!